org.eclipse.acute.SWTBotTests/pom.xml

```diff
@@ -11,6 +11,7 @@
 	<properties>
 		<testSuite>${project.artifactId}</testSuite>
 		<testClass>org.eclipse.acute.SWTBotTests.dotnetnew.AllTests</testClass>
+		<testClass>org.eclipse.acute.SWTBotTests.dotnetrun.AllTests</testClass>
```

Comment:
It's usually a better practice to have a "master" test suite, that then might invoke other TestSuites, themselves invoking TestCases. It makes it easier for people to override the `testClass` setting from command line if there is only one.
Also, I believe in this case, as a property can only contain a single value (and not multiple ones), what happens is that the last value is the one that's used and the previous gets ignored. I don't think this is your intention.